test/client_test.dart

```diff
@@ -80,7 +80,7 @@ void main() {
       await client.init();
       expect(client.isLogged(), false);
       await client.login(
-        LoginType.mLoginPassword,
```

Comment:
We could provide our own enum, that has those strings defined to not break usercode and not have to retype that everytime

Reply:
ye but do we want to? @mrmcoding88 mentioned extending those to support new login methods was a pain

Reply:
I think we should have our own defines for the officially known login methods still, yes. The functions should take a string though.

Reply:
ehh that really does not sound worth the effort :/

Reply:
if you want we could do it on the sdk side manually

Reply:
done